appinventor/components/src/com/google/appinventor/components/runtime/ToggleBase.java

```diff
@@ -0,0 +1,346 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2011-2018 MIT, All rights reserved
```

Comment:
Given that most of the code here came from Google (via Checkbox.java), I think we should retain the Google copyright here.